File: GateOne/Kimberly-gateOne/Day 7/Java/AtmMachine/Bank.java
import java.util.*;
public class Bank{

	private List <NewUserAccount> accountNumbers = new ArrayList <>();
	
	public void addAccountName(NewUserAccount name){
		this.accountNumbers.add(name);
	}
	
	public void addAccountNumber(NewUserAccount accountNumber){
		this.accountNumbers.add(accountNumber);
	}
	
	public void addAccountPin(NewUserAccount pin){
		this.accountNumbers.add(pin);
	}

	public void addAccountBalance(NewUserAccount balance){
		this.accountNumbers.add(balance);
	}

	public void removeAccountName(NewUserAccount name){
		this.accountNumbers.remove(name);
	}
	
	public void removeAccountNumber(NewUserAccount accountNumber){
		this.accountNumbers.remove(accountNumber);
	}
	
	public void removeAccountPin(NewUserAccount pin){
		this.accountNumbers.remove(pin);
	}

	public void removeAccountBalance(NewUserAccount balance){
		this.accountNumbers.remove(balance);
	}

	public String tostring(){
		return String.valueOf(accountNumbers);
	}

}

File: GateOne/Kimberly-gateOne/Day 7/javascript/Bank.js

class Bank{
    constructor(name, accountNumber, pin,){
        this.name = "";
        this.accountNumber = 0;
        this.pin = 0;
        
    }

    set name(name){
        this.name = name
    }
    get name(){
        return this.name;
    }

    set accountNumber(accountNumber){
        this.accountNumber = accountNumber
    }

    get accountNumber(){
        return this.accountNumber
    }
    set pin(pin){
        this.pin = pin
    }

    get pin(){
        return this.pin
    }

    set balance(balance){
        this.balance = balance
    }

    get balance(){
        return this.balance
    }
}